app/src/main/java/com/Executioner/meetx/MainActivity.java

```diff
@@ -5,13 +5,15 @@
 import android.annotation.SuppressLint;
 import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 
 import com.Executioner.meetx.authentication.Login_page;
 import com.Executioner.meetx.authentication.Signup_page;
 
 public class MainActivity extends AppCompatActivity {
+    String TAG="pratiksha";
```

Comment:
Always try to make meaningful TAG's, Like String TAG = "SignUp_page"

Reply:
You also wrote pratik na !

Reply:
Okay na!! that is for everyone